### PR TITLE
Synth hud CM port, synth damage overlay fix

### PR DESCRIPTION
**Pull request by Blundir**
## About The Pull Request

Ports med hud for dead and disconnected/no client/dnr synth from CM.
dreammaker_EAf1889grL

dreamseeker_bTeVB4jGR6

Also fixes synth having no damage overlay icon, Tivi accidentally broke more than a year ago, without anybody noticing.

## Why It's Good For The Game

Less confusion for medics about actual synth wellbeing, also fix good.

Synth damage overlay is now working properly.
dreamseeker_6oxNhEYZsw

## Changelog

:cl:
add: added synth hud for dead and dnr/no client/no ghost situations 
fix: fixed synth having no damage overlay icon
/:cl:

### CODE REVIEW

**Comment by FosterGrime-Heymynameslouie** [comment]
nother kewl blundir pr

spdedmhrge

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -137,7 +137,7 @@
 	if(secondary_unarmed_type)
 		secondary_unarmed = new secondary_unarmed_type()
 	if(species_flags & GREYSCALE_BLOOD)
-		brute_damage_icon_state = "greyscale"
+		brute_damage_icon_state = "grayscale"
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/human/species.dm:140]: lol

**Review by TiviPlus — APPROVED** [review]

**Comment by Losenis** [comment]
image

**Comment by Derrick2Electricboogaloo** [comment]
While you are at it, think you can figure out how to fix the damage overlay on the client side when you are dead for more than 5 minutes. (Normal DNR timer) and showing up as perma-dead while alive on minimap?

**Comment by Derrick2Electricboogaloo** [comment]
https://github.com/tgstation/TerraGov-Marine-Corps/issues/9678#issue-1145172559 this is the reported issue. I have plenty of experience with said issue.

**Comment by Blundir** [comment]
> [#9678 (comment)](https://github.com/tgstation/TerraGov-Marine-Corps/issues/9678#issue-1145172559) this is the reported issue. I have plenty of experience with said issue.

i think i know how to fix this, gonna look into it later

**Review by Kuro020 — APPROVED** [review]

**Review by Kuro020 — APPROVED** [review]

**Pull request merged**